src/index.ts

```diff
@@ -1,5 +1,23 @@
 import Delta from 'quill-delta'
-import normalizeUrl from 'normalize-url'
+import normalizeUrl, { Options as NormalizeUrlOptions } from 'normalize-url'
+import Quill from 'quill'
+
+declare global {
+  interface Window {
+    Quill: typeof Quill
```

Comment:
This should be optional, as it is not guaranteed to exist.

```ts
Quill?: typeof Quill
```

Reply:
Oh, absolutely true